pyproject.toml

```diff
@@ -31,7 +31,11 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
 # D100, D101, D102, D103: Ignore missing docstrings in tests
```

Comment:
Do we really want to ignore missing docstrings in tests? https://github.com/canonical/operator-workflows/blob/main/.github/workflows/integration_test.yaml#L245